src/token_metadatas_for_networks.ts

```diff
@@ -81,7 +82,7 @@ export const TokenMetadatasForChains: TokenMetadataAndChainAddresses[] = [
         decimals: 18,
         tokenAddresses: {
             [ChainId.Mainnet]: '0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2',
-            [ChainId.Kovan]: NULL_ADDRESS,
+            [ChainId.Kovan]: '0xac94ea989f6955c67200dd67f0101e1865a560ea',
```

Comment:
0xAaF64BFCC32d0F15873a02163e7E500671a4ffcD ?

Reply:
For context: https://developer.kyber.network/docs/Environments-Kovan/

Reply:
This one works too and is from Kyber so going with this one, thanks!